src/components/facts/list/item.js

```diff
@@ -2,43 +2,55 @@ import React, {Component} from 'react'
 
 import Icon from 'react-fa'
 
+import {spaceUrlById} from 'gEngine/space'
+import {navigateFn} from 'gModules/navigation/actions'
+import {isExportedFromSpace} from 'gEngine/facts'
 import {DistributionSummary} from 'gComponents/distributions/summary/index'
 import Histogram from 'gComponents/simulations/histogram/index'
 
-export const FactItem = ({fact, onEdit}) => (
-  <div className='Fact--outer'>
-    <div className='Fact'>
-      <div className='section-simulation simulation-sample'>
-        {_.has(fact, 'simulation.sample.values.length') && _.has(fact, 'simulation.stats.mean') &&
-          <div className='simulation-summary'>
-          <DistributionSummary
-            length={fact.simulation.sample.values.length}
-            mean={fact.simulation.stats.mean}
-            adjustedConfidenceInterval={fact.simulation.stats.adjustedConfidenceInterval}
-          />
-          </div>
-         }
-        {_.has(fact, 'simulation.sample.values.length') &&
-          <div className='histogram'>
-            <Histogram
-              height={15}
-              simulation={fact.simulation}
-              cutOffRatio={0.995}
+export const FactItem = ({fact, onEdit, isExportedFromSelectedSpace}) => {
+  const exported_from_url = `${spaceUrlById(_.get(fact, 'exported_from_id'))}?factsShown=true`
+  return (
+    <div className='Fact--outer'>
+      <div className='Fact'>
+        <div className='section-simulation simulation-sample'>
+          {_.has(fact, 'simulation.sample.values.length') && _.has(fact, 'simulation.stats.mean') &&
+            <div className='simulation-summary'>
+            <DistributionSummary
+              length={fact.simulation.sample.values.length}
```

Comment:
We should also make simulation engine functions for these accessors. We have some for values and errors, it would be good to do the same for length, mean, and the rest. It would help buffer us against any data structure changes down the line. No need to do anything in this PR, thought.